src/assets/wise5/classroomMonitor/classroomMonitorComponents/nodeGrading/nodeGradingView/nodeGradingView.ts

```diff
@@ -35,10 +37,10 @@ export class NodeGradingViewController {
   };
   sort: any;
   teacherWorkgroupId: number;
-  workgroupInViewById: any;
+  workgroupInViewById: any = {}; // whether the workgroup is in view or not
   workgroups: any;
-  workgroupsById: any;
-  workVisibilityById: any;
+  workgroupsById: any = {}; // workgroup names, statuses, scores, notifications, etc.
```

Comment:
This comment is a little confusing. The variable name is called workgroupsById so what do "workgroup names, statuses, scores, notifications, etc." have to do with it? Do you mean name, status, score, notifications are in a workgroup object?

Reply:
The original comment was 
```
this.workgroupsById = {}; // object that will hold workgroup names, statuses, scores, notifications, etc.
```
Which isn't that much more useful either. Should we just remove the comment?

Reply:
If the comment was something like

// workgroup objects contain name, status, score, notifications, etc.

it would be better. Alternatively we could just remove the comment or create an interface for the workgroup object although we may need to differentiate this workgroup object (that has had data like status, score, notification injected into it) compared to a plain workgroup object that only contains id, name, period.

Reply:
ok, I removed the comment to avoid confusion.